src/pkg/services/m365/api/lists.go

```diff
@@ -703,6 +745,23 @@ func concatenateHyperLinkFields(hyperlinkFields map[string]any) string {
 	return ""
 }
 
+func concatenateMetadataFields(metadataFieldsArr []map[string]any) string {
+	labels := make([]string, 0)
+
+	for _, md := range metadataFieldsArr {
+		mdVal, ok := md[MetadataLabelKey].(*string)
+		if ok {
+			labels = append(labels, ptr.Val(mdVal))
+		}
+	}
+
+	if len(labels) > 0 {
+		return strings.Join(labels, ",")
+	}
+
+	return ""
```

Comment:
this is redundant handling.  Join will return an empty string if the slice has 0 len.
```suggestion
	return strings.Join(labels, ",")
```